client/src/components/App/App.jsx

```diff
@@ -32,6 +32,10 @@ function App() {
     localStorage.getItem("current_lastname")
   );
   const [email, setEmail] = useState(localStorage.getItem("current_email"));
+  const [phone, setPhone] = useState(localStorage.getItem("current_phone"));
```

Comment:
Is phone a doctor specific thing? I would assume doctors and patients have some defauly phone number and hospital?

Reply:
I think phone was specific to doctors because patients do not enter their phone numbers, but this is something I could consider adding